root/layout/components/ExternalLinks.js

```diff
@@ -54,6 +54,7 @@ const ExternalLink = ({className, relationship, text}) => {
 const ExternalLinks = ({entity, empty, heading}) => {
   const relationships = entity.relationships;
   const links = [];
+  const blogLinks = [];
```

Comment:
Added a separate blogLinks because it seems like the easiest way to ensure they always sort under the official website, whatever the language.